swift/public_test.go

```diff
@@ -27,8 +26,6 @@ var _ = Suite(&PublishedStorageSuite{})
 func (s *PublishedStorageSuite) SetUpTest(c *C) {
 	var err error
 
-	rand.Seed(int64(time.Now().Nanosecond()))
```

Comment:
Might this have some implications, why was this deprecated?

Reply:
It should be seeded automatically at startup:

https://github.com/golang/go/issues/56319